packages/reports/addon/components/navi-action-list.hbs

```diff
@@ -1,103 +1,109 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<ul class="navi-action-list actions" ...attributes>
+<div class="navi-action-list flex-1" ...attributes>
 
   {{!-- Clone only enabled on a saved report --}}
-  <li class="action">
-    {{#if @item.isNew}}
-      <LinkTo
-        class="navi-reports-index__report-control clone navi-report__action-link--force-disabled"
-        disabled="true"
-        @route="reports.report.clone"
-        @model={{@item.id}}
-      >
-        <NaviIcon @icon="copy" />
-        <EmberTooltip @popperContainer="body" @text="Save report to enable clone" />
-      </LinkTo>
-    {{else}}
-      <LinkTo @route="reports.report.clone" @model={{@item.id}} class="navi-reports-index__report-control clone">
-        <NaviIcon @icon="copy" />
-        <EmberTooltip @popperContainer="body" @text="Clone the report" />
-      </LinkTo>
-    {{/if}}
-  </li>
+  {{#if @item.isNew}}
+    <LinkTo
+      class="navi-reports-index__report-control clone navi-report__action-link--force-disabled"
```

Comment:
Hmm, seems like this could be simplified, let's make an issue for this

Reply:
#442